src/containers/org/members/OrgMemberContainer.js

```diff
@@ -85,53 +92,83 @@ const OrgMemberContainer = ({
+                <CrumbLink to={membersRoute}>Members</CrumbLink>
+                <CrumbItem>{memberName}</CrumbItem>
+              </Crumbs>
+              <StackGrid gap={24}>
+                <StackGrid>
+                  <Typography variant="h1">{memberName}</Typography>
+                  <Typography>
+                    Below are all of the roles assigned to this member and data sets this member has permissions on.
+                  </Typography>
+                </StackGrid>
+                <StackGrid>
+                  <Typography variant="h2">Roles</Typography>
+                  <MemberRolesContainer
+                      member={member}
+                      organizationId={organizationId}
+                      roles={organization?.roles} />
+                </StackGrid>
+                <StackGrid>
+                  <Typography variant="h2">Data Sets</Typography>
+                  <Typography>Click on a dataset to manage permissions.</Typography>
+                </StackGrid>
+                <PermissionsActionsGrid
+                    assignPermissionsText="Add dataset"
+                    onChangeFilterByPermissionTypes={setFilterByPermissionTypes}
+                    onChangeFilterByQuery={setFilterByQuery}
+                    onClickAssignPermissions={() => setIsVisibleAddDataSetModal(true)} />
                 <DataSetPermissionsContainer
+                    filterByPermissionTypes={filterByPermissionTypes}
+                    filterByQuery={filterByQuery}
                     organizationId={organizationId}
                     onSelect={setSelection}
                     principal={memberPrincipal}
                     selection={selection} />
               </StackGrid>
-            </StackGrid>
-          </AppContentWrapper>
-        </ContentColumn>
-        {
-          selection && (
-            <PanelColumn>
-              <PermissionsPanel
-                  dataSetId={selection.dataSetId}
-                  key={`${selection.dataSetId}-${selection.permissionType}`}
-                  onClose={handleOnClosePermissionsPanel}
-                  principal={memberPrincipal}
-                  permissionType={selection.permissionType} />
-            </PanelColumn>
-          )
-        }
-      </AppContentGrid>
+            </AppContentWrapper>
+          </ContentColumn>
+          {
+            selection && (
+              <PanelColumn>
+                <PermissionsPanel
+                    dataSetId={selection.dataSetId}
+                    key={`${selection.dataSetId}-${selection.permissionType}`}
+                    onClose={handleOnClosePermissionsPanel}
+                    principal={memberPrincipal}
+                    permissionType={selection.permissionType} />
+              </PanelColumn>
+            )
+          }
+        </AppContentGrid>
+        <Modal
+            isVisible={isVisibleAddDataSetModal}
+            onClose={() => {}}
```

Comment:
For this modal, I'm passing withHeader={false} and using my own header component, so onClose is a noop because it doesn't do anything. Perhaps this should be a flow fix in LUK.